src/internet_identity/internet_identity.did

```diff
@@ -49,6 +49,11 @@ type KeyType = variant {
     seed_phrase;
 };
 
+type ProtectionType = variant {
+    protected;
```

Comment:
A comment explaining what `protected` means would be good. We could also rename this to something closer to the actual semantics like `device_required_to_modify`.